MAUI/Circular-Charts/RadialBarChart.md

```diff
@@ -0,0 +1,433 @@
+                        YBindingPath="SalesRate" 
+                        TrackFill="#FFF7ED" 
+                        TrackStrokeWidth="1"             TrackStroke="#FED7AA"  />
+    </chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+
+        RadialBarSeries series = new RadialBarSeries();
+        series.XBindingPath = "Product";
+        series.YBindingPath = "SalesRate";
+        series.TrackFill = new SolidColorBrush(Color.FromArgb("#FFF7ED"));
+        series.TrackStrokeWidth = 1;
+        series.TrackStroke = new SolidColorBrush(Color.FromArgb("#FED7AA"));
+
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+
+![Radial bar chart track fill in MAUI Chart](Chart-Types_images/maui_trackfill.png)
+
+## CenterView
+
+The view placed in the center of the radial bar chart is useful for sharing additional information about the radial bar chart. Any view can be added to the center of the radial bar chart using the [CenterView]() property of [RadialBarSeries](). The binding context of the [CenterView]() will be the respective radial bar series.
+
+### CenterHoleSize
+
+The [CenterHoleSize]() property of RadialBarSeries is used to get the diameter value of the center hole. Using the CenterHoleSize, we can protect the view in the radial bar center from overlapping with the series
+
+{% tabs %}
+
+{% highlight xaml %}
+
+    <chart:SfCircularChart>
+    <chart:RadialBarSeries ItemsSource="{Binding Data}" 
+                        XBindingPath="Product" 
+                        YBindingPath="SalesRate" 
+                        MaximumValue = 100;
+                        CapStyle = "BothCurve">
+                    <chart:RadialBarSeries.CenterView>
+                        <StackLayout HeightRequest="{Binding CenterHoleSize}"
+                                     WidthRequest="{Binding CenterHoleSize}">
+                            <Image Source="person.png" />
+                        </StackLayout>
+                </chart:RadialBarSeries.CenterView>
+    </chart:SfCircularChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+    SfCircularChart chart = new SfCircularChart();
+
+        RadialBarSeries series = new RadialBarSeries();
+        series.XBindingPath = "Product";
+        series.YBindingPath = "SalesRate";
+        series.CapStyle = CapStyle.BothCurve;
+
+        StackLayout layout = new StackLayout();
+        Image image = new Image { Source = "person.png" };
+        layout.SetBinding(HeightRequestProperty, nameof(RadialBarSeries.CenterHoleSize));
+        layout.SetBinding(WidthRequestProperty, nameof(RadialBarSeries.CenterHoleSize));
+        layout.Children.Add(image);       
+
+        series.CenterView = image;
+        chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Radial bar chart center view in MAUI Chart](Chart-Types_images/maui_radialbarchart_centerview.png)
```

Comment:
.NET MAUI Radial bar chart center view

Reply:
I have changed the  center view image alt text